### PR TITLE
📝 Improve "Config Model" documentation

### PR DESCRIPTION
**Pull request by Kludex**
Selected Reviewer: @adriangb

### CODE REVIEW

**Comment by Kludex** [comment]
please review

File: docs/usage/model_config.md [hunk 1]
```diff
@@ -97,8 +99,9 @@ See the [`ConfigDict` API documentation](../api/config.md#pydantic.config.Config
 
 ## Change behaviour globally
 
-If you wish to change the behaviour of _pydantic_ globally, you can create your own custom `BaseModel`
-with custom `model_config` since the config is inherited
+If you wish to change the behaviour of pydantic globally, you can create your own custom `BaseModel`
```
Comment (hramezani) [inline docs/usage/model_config.md:102]:
```suggestion
If you wish to change the behaviour of Pydantic globally, you can create your own custom `BaseModel`
```

File: docs/usage/model_config.md [hunk 2]
```diff
@@ -149,66 +155,184 @@ instead use the `to_lower_camel` function.
...
+1. The `age` argument is included.
+
+With `extra='forbid'`, an error is raised:
+
+```py
+from pydantic import BaseModel, ConfigDict, ValidationError
+
+
+class User(BaseModel):
+    model_config = ConfigDict(extra='forbid')  # (1)!
```
Comment (hramezani) [inline docs/usage/model_config.md:239]: Do we need `  # (1)!` here?
```suggestion
    model_config = ConfigDict(extra='forbid')
```

**Review by hramezani — APPROVED** [review]

**Comment by Kludex** [comment]
I'm going to merge it, but post reviews are also welcome.

**Pull request merged**